kedro/runner/thread_runner.py

```diff
@@ -52,10 +52,10 @@ def create_default_data_set(self, ds_name: str) -> AbstractDataSet:
         """Factory method for creating the default data set for the runner.
 
         Args:
-            ds_name: Name of the missing data set
+            ds_name: Name of the missing data set.
 
         Returns:
-            An instance of an implementation of AbstractDataSet to be used
+            An instance of an implementation of ``AbstractDataSet`` to be used
```

Comment:
How do you decide whether to use single vs double vs. no backticks? Line 31 has 1, and line 27 of parallel_runner.py has a mix of 0 and 2.

Reply:
There's no good reason for this inconsistency. Double backticks are always best because
* if the class/function/whatever being enclosed is exposed in `__init__.py` files  then it gets automatically  linked to the right page in the docs ([code that does this](https://github.com/quantumblacklabs/kedro/blob/main/docs/conf.py#L340), [click on `Pipeline` on this page for example](https://kedro.readthedocs.io/en/stable/kedro.runner.AbstractRunner.html#kedro.runner.AbstractRunner))
* if it's not a kedro object it will get formatted as code in the docs, whereas single backticks or no backticks will just do it as plain text